src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,16 +2,32 @@
 
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-    int totalRejected = 0;
+    private int totalRejected = 0;
 
+    /**
+     * Zwraca całkowitą liczbę odrzuconych wartości
+     * @return int
+     */
     public int reportRejected() {
-        return totalRejected;
-    }
-
+        return getTotalRejected();
+    } //TODO:reportRejected zwraca to samo co getTotalRejected
+    /**
+     * Dodaje wartość do tablicy jeżeli tablica nie jest pełna w przeciwnym wypadku zwiększa wartość totalRejected
+     ** @param in wartość którą chcemy dodać do listy
+     */
+    @Override
     public void countIn(int in) {
         if (!callCheck() && in > peekaboo())
-            totalRejected++;
+            setTotalRejected(getTotalRejected() + 1);

```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.